README.md

```diff
@@ -1,19 +1,41 @@

```

Comment:
I think we either need to mention you need to add the image viewer node - or remove it.
Also - do any of these images have any copyright we need to be aware of ?

Reply:
And all the images are .png - The detector works with .jpg... so they all need to be changed.

Reply:
Hey Dave (@dceejay),
I have made some changes, as you can [review](https://github.com/bartbutenaers/tfjs-coco-ssd/blob/master/README.md) on the reame page of my fork:
+ The flow has been updated to use jpg images instead of png.
+ The images (used in the http request nodes) are now all from Wikimedia commons, as indicated on the bottom of the page.
+ There is now an indication that the image viewer node should be installed.  If I would have removed that node from the flow, you can't see the input image anymore (which would turn it into a rather useless demo...